QUESTIONS.md

```diff
@@ -3,3 +3,16 @@
 What questions can you come up with?
 
 ## Cohort Questions
+
+1. React- is the router component something we can wrap our code in and break out additional components? A component within a component - Routeception.
+
+2. When calling a function within a class, are there any other rules other than you don't use the typical function syntax? Is "render" the main way this is done?
+
+3. Is React a plant, because it needs to find a 'root' to grow?? What the heck is a 'root' in React terms?
```

Comment:
Root simply is the name of the HTML component where React can generate. It doesn't have to be called Root, it could be called what ever you want it to be as long as React knows the name, and it is inside the index.html file.